CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ if(POLICY CMP0128)
   cmake_policy(SET CMP0128 NEW)
 endif()
 
-project(hwy VERSION 1.2.0)  # Keep in sync with highway.h version
+project(hwy VERSION 1.2.0 LANGUAGES CXX)  # Keep in sync with highway.h version
```

Comment:
I think the ppc builder is complaining about this. If we specify it manually, it should also include C because the try_compile seems to create a .c file.
But is this really necessary? The stackoverflow link below has another way (per-library property) to set the linker language.

Reply:
yes, not necessary